src/components/ProjectHeader/ProjectHeader.js

```diff
@@ -0,0 +1,24 @@
+import { useTranslation } from 'react-i18next';
+import './ProjectHeader.scss'
+
+
+const ProjectHeader = (props) => {
```

Comment:
V picky but any chance we could rename this to something like `ProjectIndexHeader`? It's just I found it a bit confusing as the other header is more of a project header in that it includes project name etc and is on the project show page 🤔